assets/src/containers/AssignmentPlanningV2.js

```diff
@@ -122,21 +124,31 @@ function AssignmentPlanningV2 (props) {
     settingChanged
   })
 
-  const handleAssignmentGoalGrade = (key, assignmentGoalGrade) => {
+  const handleAssignmentGoalGrade = (key, assignmentGoalGrade, prevGoalGrade) => {
+    const v = {
+      assignmentId: key,
+      assignGoalGrade: assignmentGoalGrade,
+      assignPrevGoalGrade: roundToXDecimals(prevGoalGrade, 1)
```

Comment:
Why do you round the previous goal grade here?

Reply:
otherwise it is logging it as `41.786` where are in the UI it is shown as 42

Reply:
I just felt I will log what is shown to the user. The Calculation the way it is done it is holding up to 3 digit decimal places and user setting also stored like that.

Reply:
So event logs are captured separately from user setting, I felt I can log what they see

Reply:
I gotcha, I thought the goals were rounded before being saved, rather than before being displayed.